package/rpm/SOURCES/etc/init.d/sth

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+
+# Load some fancy functions for init.d
+. /etc/rc.d/init.d/functions
+
+PARAM=$1
+
+STH_INSTANCE=${2}
+
+COMPONENT_NAME=sth
+PREFIX=/usr
+COMPONENT_DIR=${PREFIX}/${COMPONENT_NAME}
+CONFIG_FILE=${PREFIX}/${COMPONENT_NAME}/conf/sth.conf
+COMPONENT_USER=sth
+
+NODE_EXEC="/usr/bin/node"
+
+component_start()
```

Comment:
I asume this RPM does not install MongoDB (what is OK for me, it could be yet installed by other component such as Orion, and it would be shared), but makes it sense to check if MongoDB is installed? I mean, instead of running STH and waiting it crashes because no MongoDB is running, we could easily check it is running or not, and warn about that :)